tracee-ebpf/tracee/tracee.go

```diff
@@ -423,6 +430,21 @@ func New(cfg Config) (*Tracee, error) {
 			return nil, err
 		}
 
+		for _, iface := range t.config.Capture.NetIfaces[1:] {
+			ngIface = pcapgo.NgInterface{
+				Name:       iface.Name,
+				Comment:    "tracee tc capture",
+				Filter:     "",
+				LinkType:   layers.LinkTypeEthernet,
+				SnapLength: uint32(math.MaxUint16),
+			}
+
+			_, err := pcapWriter.AddInterface(ngIface)
```

Comment:
There are pros and cons to save it as a single file vs multiple files.
For example, with one file, it might be easier to correlate between events happening on different interfaces, while still being able to differentiate between the interfaces.
Let's consider this in a future PR, where we might also give an option to the user to choose to split the pcaps or not.